index.html

```diff
@@ -58,7 +58,7 @@
             <li>
               <a href="learn.html"><span class="icon-book"></span> Documentation</a>
             </li>
-            <!-- <li><a href="#"><span class="icon-beaker"></span> Developers</a></li> -->
+            <li><a href="#developers"><span class="icon-beaker"></span> Developers</a></li>
```

Comment:
Does this need to be added to other files that can be navigated to?

Reply:
Added developers link to other pages in https://github.com/json-ld/json-ld.org/pull/725/commits/a614b126dc813168b12957e259787c55b97c5493.